staging/src/k8s.io/apiserver/pkg/server/config_selfclient_test.go

```diff
@@ -62,8 +79,27 @@ func TestLoopbackHostPort(t *testing.T) {
 	if ip := net.ParseIP(host); ip == nil || !ip.IsLoopback() || ip.To4() != nil {
 		t.Fatalf("expected IPv6 host to be loopback, got %q", host)
 	}
-
 	if port != "443" {
 		t.Fatalf("expected 443 as port, got %q", port)
 	}
 }
+
+func isIPv6LoopbackSupported() (bool, bool, error) {
+	addrs, err := net.InterfaceAddrs()
+	if err == nil {
```

Comment:
nit: can we flip the err check here? Easier to read IMO 
```
if err != nil {
     return false, false, err
}
....
....
return ipv6, ipv6 && !ipv4, nil
```

Reply:
+1

Reply:
fixed